x/version/chains.go

```diff
@@ -0,0 +1,25 @@
+package version
+
+const (
+	MochaChainID          = "mocha"
+	ArabicaChainID        = "arabica-4"
+	BlockspaceraceChainID = "blockspacerace-0"
+	MainnetChainID        = "celestia-1"
+)
+
+// StandardChainVersions returns a map of chain IDs to their respective
+// ChainVersionConfig. Each hardfork should be added to this map.
+func StandardChainVersions() map[string]ChainVersionConfig {
+	version0Only := NewChainVersionConfig(map[uint64]int64{
+		0: 0,
```

Comment:
This is `0` because the current network is set to `0` right?

Reply:
yeah, hopefully we can stay 1:1 with our major version for simplicity